### DISCUSSION TITLE
gothic-win32-v1.0.1035

### DISCUSSION THREAD

**Discussion by Try** [Try/OpenGothic#137] [General]
![изображение](https://user-images.githubusercontent.com/122894/115162319-1f4a2b80-a0a3-11eb-809a-81a829348019.png)

* General
	* Added names for save-games
	* Adjust behavior of AI_ProcessInfo, to stop walking animation before dialog
	* Optimize ZenLib to improve game load-time
	* Added oCMobFire support - city lighting now works
	* Initial torch support
	
* Graphics
	* Added morphing animation
	* Major progress on shadowmaps: shadows-pass now faster and image is more detailed
	* Improve camera collision
	* Particle-fx trails initial support

* Script-api
	* Bugfix: initialize HERO, before calling any scripts(initScripts)
	* Added dedicated overlay queue ("For the king!" quote now works)
	
* UI
	* Multiple UI adjustments, to have consistency with original game (#135)

![изображение](https://user-images.githubusercontent.com/122894/115162325-26713980-a0a3-11eb-8aa6-f29006043d26.png)
![изображение](https://user-images.githubusercontent.com/122894/115162339-47398f00-a0a3-11eb-91a2-cbc307440088.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/Try/OpenGothic/releases/tag/gothic-win32-v1.0.1035'>gothic-win32-v1.0.1035</a>.</em>

**Comment by LeksiDor** [comment]
First of all, thank you so much for this project! That was the only way I could run Gothic 2 on Windows 10, although it's Steam version. My run was done on release v1.0.911. I am very familiar with this game, and this time decided to play as Dexterity-based Paladin bowman. Unfortunately though, there were some game-breaking experiences during the gameplay. You can find them below.

Addon world:
* Ladded (in the swamp) is not interactible (already reported).
* Temple of Adanos: Rademes does not appear in the final trap room with floor spikes.
* Cannot access normally under-water cave near to Malcom: when try to swim under-water through the passage, just fall onto the lake bottom as if there was no water at all.

Khorinis:
* Secret library room in the monastery, to where the Paladin player creates the teleport rune. First, Paladin's heavy armor does not appear on the table (although I finished all previous quests). Second, the room chest does not open, with any lock-picking combination.

General:
* Pick-axes weirdly disappear on attempt to use.
* Beliar claw does not seem to work constantly.
* Acrobatics animation is weird when you press "jump" without release.
* Acrobatics animation stops when your jump continues below the level of initial jumping point.
* When you use a teleport rune, the blue glow does not disappear and follows you all the time.
* Rings are being grouped in the inventory, which means you cannot equip two same rings.

**Reply by Try** [reply]
Hi, @LeksiDor and thanks for feedback!

Those issue suppose to be fixed in 1035:
* Cannot access normally under-water cave
* Beliar claw does not seem to work constantly. - AFAIK in 911 the claw do damage, but visual was absent. 
* When you use a teleport rune, the blue glow does not disappear and follows you all the time.

Secret library room in the monastery: 
* Not 100% sure, but armor suppose to be inserted right at chapter 5 starts. Because `HERO` was not initialized, game script was not aware of your guild. Should be fixed now.
